Firmware/Test Sketches/WebServer/WebServer.ino

```diff
@@ -0,0 +1,162 @@
+/*
+  Web server test program
+*/
+
+#include <ESPAsyncWebServer.h>  //Get from: https://github.com/me-no-dev/ESPAsyncWebServer
+#include <SdCardServer.h>       //Get v1.0 from: https://github.com/LeeLeahy2/SdCardServer
+#include <SPI.h>
+#include <pgmspace.h>
+#include <WiFi.h>
+
+#include "SdFat.h" //http://librarymanager/All#sdfat_exfat by Bill Greiman. Currently uses v2.1.1
+
+#define ASCII_LF        0x0a
+#define ASCII_CR        0x0d
+
+int keyIndex = 0;
+char password[1024];          // WiFi network password
+char ssid[1024];              // WiFi network name
+
+typedef struct struct_online {
+  bool microSD = false;
+} Online;
+
+Online online;
+SdFat sd;
+int sdCardMounted;
+AsyncWebServer server(80);
+int status = WL_IDLE_STATUS;  // the Wifi radio's status
+int wifiBeginCalled;
+int wifiConnected;
+
+//-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+int mountSdCard(void) {
```

Comment:
This is purely stylistic, but sketches have setup(), then loop(), then any supporting functions. Users (I constantly think about the students I taught) want to know what the thing does, and can get lost scrolling though lots of top functions just to get to setup/loop.